addon/helpers/has-question-type.js

```diff
@@ -0,0 +1,14 @@
+import { helper } from "@ember/component/helper";
+import { dasherize } from "@ember/string";
+
+export default helper(function hasQuestionType(
+  [obj, ...expected],
+  { replace = "Question" }
+) {
+  const typename = dasherize((obj.__typename || "").replace(replace, ""));
+  const typenames = expected.map((typename) =>
+    dasherize(typename.replace(replace, ""))
+  );
```

Comment:
Nitpick: This might become a little more readable if you extract the `dasherize(... replace())` into a little helper function.

Reply:
`dasherizeAndReplace` or what? I really don't think that would help at all... This is very basic string alternation, shouldn't be too complicated for anyone to understand.

Reply:
As discussed, done